.github/workflows/WORKFLOWS.md

```diff
@@ -0,0 +1,11 @@
+# GitHub Actions Workflows
+
+| Workflow Name | Filename | Description | Required | Triggers |
+| -- | -- | -- | -- | -- |
+| Generate JSON Schema*     | generate_jsonschema.yaml      | Generate the JSON Schema and create PR | Y | Push:main (*) |
+| Check Unexpected Changes* | check_unexpected_changes.yaml | Notify contributor that PR may change the JSON Schema generation | Y | PR (*) |
+| Client Smoke Tests        | test_clients.yaml             | Notify contributor that PR may change client generation | N | PR (proto/* \| third_party/* \| testdata/* \| workflow) |
+| Test Proto Syntax         | test_proto_syntax.yaml        | Validate proto syntax through compliation | Y | PR (proto/* \| third_party/* \| scripts/* \| tools/* \| workflow) |
+| Validate JSON Schemas     | validate_jsonschemas.yaml     | Validate JSON Schema format | Y | PR (jsonschema/* \| scripts/* \| tools/* \| workflow) |
```

Comment:
Validate isn't one of the verbs we're discussing, should this be renamed?

Reply:
done